### DISCUSSION TITLE
UnhandledPromiseRejectionWarning: unexpected token at "&gt;

### DISCUSSION THREAD

**Discussion by rickyplc** [harttle/liquidjs#322] [Q&A]
I am trying to use the greater than operator `>` in a simple if condition `{% if my_variable > 1 %}s{%  endif %}` but this breaks. This is happening with the less than operator too `<`.

This in my view is a major issue, any ideas on how to fix this?

**Comment by harttle** [comment]
It's OK in the playground: https://liquidjs.com/playground.html#eyUgaWYgbXlfdmFyaWFibGUgPiAxICV9eWVzeyVlbHNlJX1ub3slIGVuZGlmICV9

I believe you're escaping the template content before pass it to liquidjs. Try print out the template string before parse and render.

**Reply by rickyplc** [reply]
sorry, I have only just seen your message. I will try this. Thank you!

**Reply by rickyplc** [reply]
That was the problem, thanks!
